staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -18,6 +26,12 @@
           <mesh filename="package://staubli_rx160_support/meshes/rx160/collision/base_link.stl"/>
         </geometry>
       </collision>
+      <inertial>
+          <mass value="${base_mass}"/>
+          <inertia ixx="0.001" ixy="0.0" ixz="0.0"
+                   iyy="0.001" iyz="0.0"
+                   izz="0.001"/>
+        </inertial>
```

Comment:
I got the actual values extracted from the official Staubli CAD models using Solidworks. I tested/checked moments of inertia in Gazebo.